Library/PaginateTests.swift

```diff
@@ -11,12 +11,13 @@ final class PaginateTests: TestCase {
   let requestFromCursor: (Int) -> SignalProducer<[Int], Never> = { c in .init(value: c <= 2 ? [c] : []) }
   let valuesFromEnvelope: ([Int]) -> [Int] = id
   let cursorFromEnvelope: ([Int]) -> Int = { ($0.last ?? 0) + 1 }
+  let statsFromEnvelope: ([Int]) -> Int = { _ in return 10 }
```

Comment:
Why are you returning 10? This should probably be `$0.count`?

Reply:
10 is an arbitrary number I chose. Returning `$0.count` would return the count of results for a page during pagination, but stats is supposed to be the same regardless of the requested page.